src/main/java/seedu/address/model/person/Deadline.java

```diff
@@ -0,0 +1,70 @@
+package seedu.address.model.person;
```

Comment:
I know you mentioned it, but for the purpose of the PR, should we rename the person package to task?